clients/osv.go

```diff
@@ -56,7 +56,9 @@ func (v osvClient) ListUnfixedVulnerabilities(
 
 	response := VulnerabilitiesResponse{}
 
-	if err == nil { // No vulns found
+	// either no vulns found, or no packages detected by osvscanner, which likely means no vulns
+	// while there could still be vulns, not detecting any packages shouldn't be a runtime error.
+	if err == nil || errors.Is(err, osvscanner.NoPackagesFoundErr) {
```

Comment:
@another-rex is there anything big I'm missing with this assumption?

As written, this change would give empty repos a 10/10 for Vulnerabilities, which would be reasonable. The other option would be returning an inconclusive score (`-1`).

Reply:
Chatted with Laurent, this is already something that could occur, so we are going to merge as written.

If osv-scanner detects 1 vuln-free package, and fails to detect other packages we wouldn't know.

Reply:
No, this assumption is correct!